### PR TITLE
feat #115246: WBO ItemView initial layout

### PR DESCRIPTION
**Pull request by ChrisSkj**
### What's new?

- Rename a bunch of stuff
- Add new types for WBO details
- Add tabs fo WBO details

image
image

### CODE REVIEW

File: src/layers/external/wbo/ui/itemView/WBOItemView.module.css [hunk 1]
```diff
@@ -0,0 +1,23 @@
+.links {
+  display: flex;
+  justify-content: end;
+  margin-top: 25px;
+  margin-bottom: 15px;
+}
+
+.link {
```
[the remark this answers is not in the document]
Comment (ErlendAHall) [reply src/layers/external/wbo/ui/itemView/WBOItemView.module.css:8]: Actually, those links are the same as "BlackLink" which you import from Echo Components. Use them instead, the styling should be good there

File: src/layers/external/wbo/ui/itemView/tabs/WBOTabs.tsx [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+    const tabListContainer = orientationVertical
+        ? styles.tabsRow
+        : styles.tabsRowHorizontal;
+
+    return (
+        <Tabs
+            className={tabsContainer}
+            activeTab={activeTab}
+            onChange={switchTab}
+        >
```
Comment (ErlendAHall) [inline src/layers/external/wbo/ui/itemView/tabs/WBOTabs.tsx:99]: Noticed the tab scroll buttons are not vertically aligned

**Comment by olsoybakk** [comment]
Add scroll on horizontal- and fullscreen view?
image

**Review by olsoybakk — APPROVED** [review]
approved! - but remember our comments 😇

File: src/layers/external/wbo/ui/itemView/tabs/WBOTabs.tsx [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+                delayToScrollEnd = setTimeout(() => {
+                    cachedList.scrollLeft === 0
+                        ? setPrevDisabled(true)
+                        : setPrevDisabled(false);
+                    containerWidth + Math.ceil(cachedList.scrollLeft) ===
+                    totalWidth
+                        ? setNextDisabled(true)
+                        : setNextDisabled(false);
+                }, 20);
+            };
```
[the remark this answers is not in the document]
Comment (ChrisSkj) [reply src/layers/external/wbo/ui/itemView/tabs/WBOTabs.tsx:51]: Made a bug on this

**Pull request merged**